pkg/shell/hosts_dialog.jsx

```diff
@@ -373,7 +377,10 @@
                        </Button>
                    </>}
             >
-                {body}
+                <Stack hasGutter>
+                    { this.props.dialogError && <ModalError dialogError={this.props.dialogError} />}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17221-20221201-102811-6d301c18-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)